packages/vaadin-lumo-styles/components/combo-box.css

```diff
@@ -0,0 +1,32 @@
+@import './input-container.css';
```

Comment:
Let's add license header comment to all new CSS files in this folder.

Reply:
Done.